### PR TITLE
Feature/configuration files

### PR DESCRIPTION
**Pull request by lee-benjamin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,14 +11,22 @@
     <properties>
       <twitter4j.version>[4.0,)</twitter4j.version>
       <maven.shade.version>3.1.0</maven.shade.version>
+      <dropwizard.version>1.2.2</dropwizard.version>
+      <maven.compiler.source>1.8</maven.compiler.source>
```
Comment (kev-chan) [inline pom.xml]: you can combine this version with the one below it if you like, this fine too however

File: src/main/java/com/benjamin/benleethium/BenLeethiumApplication.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import io.dropwizard.Application;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import com.benjamin.benleethium.resources.BenLeethiumResource;
+import com.benjamin.benleethium.health.TemplateHealthCheck;
+import twitter4j.TwitterFactory;
+import twitter4j.conf.ConfigurationBuilder;
+
+public class BenLeethiumApplication extends Application<BenLeethiumConfiguration> {
+    public static TwitterFactory twitterFactory;
```
Comment (kev-chan) [inline src/main/java/com/benjamin/benleethium/BenLeethiumApplication.java]: static variables accessed in other areas generally aren't good practices

File: src/main/java/com/benjamin/benleethium/api/ErrorResponse.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package com.benjamin.benleethium.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ErrorResponse {
+
+    private String message;
+
+    public ErrorResponse() {
```
Comment (kev-chan) [inline src/main/java/com/benjamin/benleethium/api/ErrorResponse.java]: you can remove the default constructor

File: src/main/java/com/benjamin/benleethium/api/GetResponse.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class GetResponse {
+
+    private List<String> content;
+
+    public GetResponse() {
```
Comment (kev-chan) [inline src/main/java/com/benjamin/benleethium/api/GetResponse.java]: same as above

File: src/main/java/com/benjamin/benleethium/api/PostResponse.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
+package com.benjamin.benleethium.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PostResponse {
+
+    private String content;
+
+    public PostResponse() {
```
Comment (kev-chan) [inline src/main/java/com/benjamin/benleethium/api/PostResponse.java]: same as above

File: src/main/java/com/benjamin/benleethium/BenLeethiumApplication.java (hunk 2 above)
Comment (lee-benjamin) [reply src/main/java/com/benjamin/benleethium/BenLeethiumApplication.java]: ok; will fix with singleton pattern or something in a different lab.

**Pull request merged**